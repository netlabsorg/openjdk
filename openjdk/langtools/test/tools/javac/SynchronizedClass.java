/* @test  /nodynamiccopyright/
   @bug 4037020 4785453
   @summary Verify that ClassModifier "synchronized" is not allowed.
   @author dps

   @run shell SynchronizedClass.sh
*/

public synchronized class SynchronizedClass { } // ERROR
