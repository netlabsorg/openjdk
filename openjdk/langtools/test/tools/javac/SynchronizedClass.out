SynchronizedClass.java:9: modifier synchronized not allowed here
public synchronized class SynchronizedClass { } // ERROR
                    ^
1 error
